### PR TITLE
add icons for the layouts in svg format

### PR DESCRIPTION
**Pull request by 888yalo888**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,3 @@
 VITE_POCKET_BASE_URL='http://0.0.0.0:8090'
 
-// VITE_POCKET_BASE_URL='https://dnd-helper.pockethost.io'
+VITE_POCKET_BASE_URL='https://dnd-helper.pockethost.io'
```
Comment (DenielNCore) [inline .env:3]: revert this change for now

File: src/components/SVGIcon.vue [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  );
+
+  const Icon = computed(() => {
+    switch (props.type) {
+      case Type.clipboard:
+        return Clipboard;
+      case Type.listUl:
+        return ListUl;
+      case Type.tableCells:
+        return TableCell;
```
Comment (DenielNCore) [inline src/components/SVGIcon.vue:22]: switch must have default:
        return null;

File: src/types/iconTypes.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+enum Type {
+    clipboard = 'clipboard',
+    listUl = 'listUl',
+    tableCells = 'tableCells',
```
Comment (DenielNCore) [inline src/types/iconTypes.ts]: need to add none = 'none', value

File: src/components/SVGIcon.vue [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  import ListUl from '@/assets/list-ul-solid.svg';
+  import TableCell from '@/assets/table-cells-large-solid.svg';
+  import { computed } from 'vue';
+  import { Type } from '@/types/iconTypes';
+
+  const props = withDefaults(
+    defineProps<{
+      type: Type;
+    }>(),
+    { type: Type.clipboard },
```
Comment (DenielNCore) [inline src/components/SVGIcon.vue]: default value should be something else like Type.none (if you'll add such)

File: .env (hunk 1 above)
Comment (888yalo888) [reply .env:3]: I didn't touch this file

Comment (DenielNCore) [reply .env:3]: you did :-)

**Review by DenielNCore — APPROVED** [review]
